lib/guardian_db/token.ex

```diff
@@ -35,7 +31,8 @@ defmodule GuardianDb.Token do
   def find_by_claims(claims) do
     jti = Map.get(claims, "jti")
     aud = Map.get(claims, "aud")
-    GuardianDb.repo.get_by(Token, jti: jti, aud: aud)
+
+    GuardianDb.repo().get_by(query_schema(), jti: jti, aud: aud)
```

Comment:
All the queries have to use `query_schema` because it will return a tuple, no just the name.

Example: `{"table_name", Token}`